content/03.results.md

```diff
@@ -2,47 +2,44 @@
 
 ### Molecular Subtyping of OpenPBTA CNS Tumors
 
 Over the past two decades, experts in neuro-oncology have worked with the WHO to iteratively redefine the classifications of central nervous system (CNS) tumors [@pubmed:11895036; @doi:10.1007/s00401-007-0243-4].
-More recently, in 2016 [@doi:10.1007/s00401-016-1545-1], molecular subtypes driven by genetic alterations have been integrated into these classifications.
-In 2011, the Children’s Brain Tumor Tissue Consortium, now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
-Since the CBTN opened its collection protocol in 2011, before molecular data were integrated into classifications, the majority of the tumors within the OpenPBTA lacked molecular subtype annotations at the time of tissue collection.
-Moreover, the OpenPBTA data does not yet feature methylation arrays which are increasingly used to inform molecular subtyping and cancer diagnosis.
+In 2016 [@doi:10.1007/s00401-016-1545-1], molecular subtypes driven by genetic alterations have been integrated into these classifications.
+Since CBTN specimen collection began in 2011 before molecular data were integrated into classifications, the majority of OpenPBTA tumors lacked molecular subtype information at the time of tissue collection.
+Moreover, the OpenPBTA does not yet feature methylation arrays which are increasingly used to inform molecular subtyping and cancer diagnosis.
```

Comment:
tricky - as we _just_ performed this on 1800 tumors, but did not want to integrate here last minute...so, we can say PBTA for now since hopefully is published soon!